infrastructure/protocol-upgrade/README.md

```diff
@@ -249,5 +249,6 @@ $ zk f yarn start transactions cancel-upgrade \
 --zksync-address <zksyncAddress> \
 --gas-price <gas-price> \
 --nonce <nonce> \
---environment <environment>
+--environment <environment> \
+--new-governance <governanceAddress>
```

Comment:
let's move it to the place before environment. 
I think it's a bit better, because we have all eth related stuff in one place